FRPs/proposal-2-liquidations.md

```diff
@@ -0,0 +1,50 @@
+---
+id: 2
+title: MEV and Liquidations
+team: Kaihua Qin, Liyi Zhou, Arthur Gervais
+created: 2020-12-10
+paper: 1
+---
+
+# Summary
+With over 6B USD total value locked for the top 3 lending protocols alone, this use case has grown into a substantial DeFi market. Lending entails the risks of a diminishing collateral value, which may render debt unhealthy. Unhealthy debt is typically resolved through liquidations, a process of selling collateral at a discount to liquidators.
+
+One dominant liquidation design is the fixed spread liquidation mechanism, used by Aave, Compound and dYdX. Interestingly, the fraction of the collateral that a liquidator can purchase at a discount amounts to 50% for Aave and Compound, and even 100% for dYdX (commonly referred to as close factor). It therefore appears that the fixed spread liquidation mechanisms are benefitting the liquidators more than the borrowers.
+
+Drawing the link to MEV, each liquidation event may qualify as Miner Extractable Value (MEV). For example, in a recent price oracle manipulation attack, a liquidator performed an oracle price post and liquidated millions of USD within one transaction, yielding a profit of nearly 4M USD. 
```

Comment:
https://etherscan.io/tx/0x53e09adb77d1e3ea593c933a85bd4472371e03da12e3fec853b5bc7fac50f3e4

Reply:
Small note, but we don't know whether that transaction was a manipulation attack, right? All we know is the price of DAI on Coinbase rose sharply. But there are some innocuous explanations for this too, e.g. someone panic buying DAI to prevent a CDP liquidating as the price of ETH fell.

Reply:
Indeed, we explored this at the time with @tinchoabbate, and concluded no manipulation took place. Compound allows for any actor to update the price as long as it's coming from their trusted feed (currently only Coinbase). @Austin-Williams has some thoughts on this too I believe.

That said, this is still an interesting MEV extraction event.

Reply:
So, that was _not_ a price oracle manipulation attack. That's normal behavior for Compound.
Compound allows anyone to submit the price data (signed by Coinbase) on chain. If the new price allows an unhealthy borrower to be liquidated, then there is race to submit that price and perform the liquidation in the same transaction, resulting in a priority gas auction to see who gets to do it.

Reply:
FWIW, I think fixed-discount (or "fixed spread") liquidation is a much safer design pattern (from a security and reliability perspective) than auctions.

While auctions are more efficient and cheaper for the borrower being liquidated, they are more risky for the lenders (because they have a larger attack surface) and they are less capital efficient for liquidators (because they can't flash-borrow the capital they need to liquidate).

So, on the whole, I tend to urge people to use the fixed-discount pattern.

But I agree that it does result in more MEV than auctions.